### DISCUSSION TITLE
Github support : Reinstatement Request #2983252 no response for two months from 4 September 2024

### DISCUSSION THREAD

**Discussion by devaapriana** [community/community#142231] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Two months ago my account with username **codemysister** was hacked and I managed to log in again but my GitHub account had a spam flag, I deleted the potentially spam repository and activated 2FA, then on September 4 2024 I created a ticket to recover my account from the spam flag with the code # 2983252, but until now there has been no response, I am currently having difficulty applying for jobs because my github cannot be seen by other people. I ask for help on how to get my account back to normal.

![screencapture-support-github-ticket-personal-0-2983252-2024-10-22-13_20_05](https://github.com/user-attachments/assets/a62016e4-dd72-4e5a-b6db-e8329022501a)

**Comment by afindo** [comment]
same as me

**Comment by eric-gitta-moore** [comment]
same here. ticket #3090267
